exporter/type.go

```diff
@@ -33,3 +42,19 @@ type (
 		ComponentType string `yaml:"componentType"`
 	}
 )
+
+func (s *StaticConfig) Validate() error {
+	for _, cluster := range s.Clusters {
+		for _, instance := range cluster.Instances {
+			switch instance.ComponentType {
+			case ComponentTypeGraphd, ComponentTypeMetad, ComponentTypeStoraged, ComponentTypeMetaListener, ComponentTypeStorageListener, ComponentTypeDrainer:
```

Comment:
How about add a var like `var supportedComponentType map[string]struct{}`?

Reply:
good idea, but there won't be many instances in one exporer, so I think it's optional

Reply:
Yes, so I approved.